rawdisk/scheme/mbr.py

```diff
@@ -47,6 +56,43 @@ def part_offset(self):
     def part_type(self):
         return self.fields.part_type
 
+    def chs2lba(self, cyl, head, sect, geometry=DEFAULT_GEOMETRY):
+        """ helper for consistency check """
+        hpc, spt = geometry['HPC'], geometry['SPT']
+        return sect-1 + head*spt + cyl*hpc*spt
+
+    def get_type_label(self):
+        if self.fields.part_type in TYPES:
+            return '%s' % TYPES[self.fields.part_type]
+        else:
+            return 'unknown'
+
+    def __str__(self):
+        # calculated values
+        fields = self.fields.copy()
+        fields['chs_start_sector'] = self.chs2lba(
```

Comment:
These are not properties, but calculations. I grouped them into dict just to pass to format() function. Those calculations are only useful for checking consistency of the record, but I'd like to skip adding too much stuff in on PR. Printing the info was enough for my use case.

The thing that you may be interested to test is the conversion function `chs2lba`, which is already exposed.